src/main/java/io/logz/guice/jersey/configuration/WebsocketConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package io.logz.guice.jersey.configuration;
+
+import java.util.*;
+
+public class WebsocketConfiguration {
+
+    private List<Class<?>> endpointClasses = new ArrayList<>();
+
+    public List<Class<?>> getEndpointClasses() {
+        return Collections.unmodifiableList(endpointClasses);
+    }
+
+    public WebsocketConfiguration withEndpointClass(Class<?> endpointClass) {
```

Comment:
I think it's better to call this method `addEndpointClass`,
Also, `withEndpointClasses` --> `addEndpointClasses `

I'm using the word `with` when the user should call this method once and `add` when the user can call it multiple times